test/recipes/02-test_errstr.t

```diff
@@ -57,7 +57,18 @@ plan tests => scalar @Errno::EXPORT_OK
 foreach my $errname (@Errno::EXPORT_OK) {
     # The error names are perl constants, which are implemented as functions
     # returning the numeric value of that name.
-    &ok(match_syserr_reason("Errno::$errname"->()))
+    my $errcode = "Errno::$errname"->();
+
+    # On most systems, there is no E macro for errcode zero in <errno.h>,
+    # which means that it seldom comes up here.  However, reports indicate
+    # that some platforms do have an E macro for errcode zero.
+    # With perl, errcode zero is a bit special.  Perl consistently gives
+    # the empty string for that one, while the C strerror() may give back
+    # something else.  The easiest way to deal with that possible mismatch
+    # is to skip this errcode.
+    continue if $errcode == 0;
```

Comment:
This line is causing the number of tests executed to mismatch the expected number. So instead, the following could be used:
```
    if ($errcode == 0) {
        &ok(1) # Ignore this error code
    } else {
        &ok(match_syserr_reason($errcode))
    }
```